pkg/applications/helmclient/testdata/examplechart-git/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
What's the point of this file? Do we expect folks to create a Mercurial repository here?

I am personally strictly against these "it's a best-practice gitignore file, I found on GitHub and it says it's the standard and everyone is supposed to use it, it's a community thing" gitignore files. They just grow and grow and grow and everyone stuffs their random stuff in their and they keep growing and growing and accumulating nonsense. Who decided that it's exactly these 3 IDEs that are included here?